### PR TITLE
chore: Copyright and license for snippets

### PR DESCRIPTION
**Pull request by dellagustin-sap**
This commit adjusts the copyright and license information for files containing snippets reused or derived from
https://github.com/sapmentors/cds-pg.

### CODE REVIEW

**Comment by dellagustin-sap** [comment]
Tested with `docker run --rm --volume $(pwd):/data fsfe/reuse lint`.
Results summary:
```
# SUMMARY

* Bad licenses: 0
* Deprecated licenses: 0
* Licenses without file extension: 0
* Missing licenses: 0
* Unused licenses: 0
* Used licenses: MIT, Apache-2.0
* Read errors: 0
* files with copyright information: 306 / 306
* files with license information: 306 / 306

Congratulations! Your project is compliant with version 3.0 of the REUSE Specification :-)
```

**Comment by dellagustin-sap** [comment]
Hello @SebastianWolf-SAP , could you please review how I set the copyright and license for the snippets coming from https://github.com/sapmentors/cds-pg?
Initially I wanted to keep everything licensed as Apache-2.0 and somehow document that those snippets were initially MIT, but I could not find how to do that. REUSE would complain about an unused license and the result would be non-compliant. So, at the end I kept those files licensed as MIT, which seems fine to me.
@johannes-vogel , would that be ok for your team as well?

**Review by SebastianWolf-SAP — APPROVED** [review]
LGTM :)

File: .reuse/dep5 [hunk 1]
```diff
@@ -27,3 +27,14 @@ Disclaimer: The code in this project may include calls to APIs ("API Calls") of
...
+Files:
+ postgres/test/beershop/db/init/beershop.sql
+ postgres/test/odata-string-functions.test.js
+ postgres/test/ql.test.js
+ postgres/test/service.test.js
+Copyright: 
+ 2022 SAP SE or an SAP affiliate company and cds-dbs contributors
+ Copyright (c) 2020 SAP Mentors & Friends
+License: MIT
+Comment: The content of these files, entirely or in part, comes from https://github.com/sapmentors/cds-pg, which is MIT licensed.
```
Comment (patricebender) [inline .reuse/dep5:40]:
```suggestion
Comment: The content of these files, entirely or in part, comes from https://github.com/sapmentors/cds-pg, which is MIT licensed.

```

File: .reuse/dep5 [hunk 2]
```diff
@@ -27,3 +27,14 @@ Disclaimer: The code in this project may include calls to APIs ("API Calls") of
 Files: *
 Copyright: 2022 SAP SE or an SAP affiliate company and cds-dbs contributors
 License: Apache-2.0
+
+Files:
+ postgres/test/beershop/db/init/beershop.sql
```
Comment (patricebender) [inline .reuse/dep5:32]:
```suggestion
```

**Review by patricebender — APPROVED** [review]

File: .reuse/dep5 [hunk 3]
```diff
@@ -27,3 +27,13 @@ Disclaimer: The code in this project may include calls to APIs ("API Calls") of
 Files: *
 Copyright: 2022 SAP SE or an SAP affiliate company and cds-dbs contributors
 License: Apache-2.0
+
+Files:
+ postgres/test/odata-string-functions.test.js
```
Comment (johannes-vogel) [inline .reuse/dep5:32]:
```suggestion
 postgres/test/beershop/*
 postgres/test/odata-string-functions.test.js
```

File: LICENSES/MIT.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+MIT License
```
[the remark this answers is not in the document]
Comment (dellagustin-sap) [reply LICENSES/MIT.txt:1]: Hello @johannes-vogel ,

Do you meant, is it necessary to include the `MIT.txt` license in the `LICENSES` folder, even when we had already mentioned it in the `.reuse/dep5` file (https://github.com/cap-js/cds-dbs/blob/7cc13b9e9c4e574bfc334d1ceb860578014e797b/.reuse/dep5#L38)?

If yes, my understanding is that it has be included due to the requirement from the MIT license itself stated here: https://github.com/cap-js/cds-dbs/blob/7cc13b9e9c4e574bfc334d1ceb860578014e797b/LICENSES/MIT.txt#L7

Additionally, the REUSE tool would "complain" if the license file is not included (I tested):
```
# MISSING LICENSES

'MIT' found in:
* postgres/test/beershop/db/init/beershop.sql
* postgres/test/odata-string-functions.test.js
* postgres/test/ql.test.js
* postgres/test/service.test.js



# SUMMARY

* Bad licenses: 0
* Deprecated licenses: 0
* Licenses without file extension: 0
* Missing licenses: MIT
* Unused licenses: 0
* Used licenses: Apache-2.0, MIT
* Read errors: 0
* files with copyright information: 306 / 306
* files with license information: 306 / 306

Unfortunately, your project is not compliant with version 3.0 of the REUSE Specification :-(
```

I'll leave it for you to "resolve" the conversation, if this answers your question.

**Review by johannes-vogel — APPROVED** [review]

**Pull request merged**